buildSrc/subprojects/configuration/src/main/kotlin/build-extensions.kt

```diff
@@ -24,34 +24,34 @@ import org.gradle.kotlin.dsl.extra
 
 @Suppress("unchecked_cast")
 val Project.libraries
-    get() = rootProject.extra["libraries"] as Map<String, Map<String, String>>
+    get() = if (rootProject.extra.has("libraries")) rootProject.extra["libraries"] as Map<String, Map<String, String>> else mapOf()
```

Comment:
Need this, for pre-compiled script plugin compilation. (In a next step we shall get rid of using `extra` altogether.)